### DISCUSSION TITLE
Title: Want to update the projects name and number for all domains in `readme.md` files .

### DISCUSSION THREAD

**Discussion by Kushal997-das** [Kushal997-das/Project-Guidance#1142] [Ideas]
### Explain what you want to add/change in documentation.

Whoever wants to work in this can tell.

**Comment by abhi-la-sha** [comment]
Hii @Kushal997-das , I'd like to make changes in the documentation part.
1. AI folder --> Intermediate--> Hand gestures . It has 2 readme.md files one of which is empty
2. Blockchain folder in the Snd_Ether folder there are 2 readme.md files one of which is empty
  There are many more such issues to be fixed.

Please assign this to me.

**Comment by Kushal997-das** [comment]
@abhi-la-sha This issue is regarding the updated project name and number from all domains and add in `readme.md` file.
